[2025-02-01T02:05:22.810+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_to_bigquery_dag.upload_to_bigquery manual__2025-02-01T02:00:15.984417+00:00 [queued]>
[2025-02-01T02:05:22.815+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_to_bigquery_dag.upload_to_bigquery manual__2025-02-01T02:00:15.984417+00:00 [queued]>
[2025-02-01T02:05:22.815+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2025-02-01T02:05:22.822+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): upload_to_bigquery> on 2025-02-01 02:00:15.984417+00:00
[2025-02-01T02:05:22.825+0000] {standard_task_runner.py:57} INFO - Started process 1500 to run task
[2025-02-01T02:05:22.827+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'gcs_to_bigquery_dag', 'upload_to_bigquery', 'manual__2025-02-01T02:00:15.984417+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/json_to_bigquery.py', '--cfg-path', '/tmp/tmp28jfesd6']
[2025-02-01T02:05:22.829+0000] {standard_task_runner.py:85} INFO - Job 117: Subtask upload_to_bigquery
[2025-02-01T02:05:22.875+0000] {task_command.py:410} INFO - Running <TaskInstance: gcs_to_bigquery_dag.upload_to_bigquery manual__2025-02-01T02:00:15.984417+00:00 [running]> on host 3f33197bb456
[2025-02-01T02:05:22.939+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_to_bigquery_dag' AIRFLOW_CTX_TASK_ID='upload_to_bigquery' AIRFLOW_CTX_EXECUTION_DATE='2025-02-01T02:00:15.984417+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-01T02:00:15.984417+00:00'
[2025-02-01T02:05:22.945+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-02-01T02:05:22.953+0000] {logging_mixin.py:150} INFO - üî¥ No data to insert into BigQuery. Î≥ÄÌôòÎêú Îç∞Ïù¥ÌÑ∞Í∞Ä ÏóÜÏäµÎãàÎã§!
[2025-02-01T02:05:22.953+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-01T02:05:22.958+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=gcs_to_bigquery_dag, task_id=upload_to_bigquery, execution_date=20250201T020015, start_date=20250201T020522, end_date=20250201T020522
[2025-02-01T02:05:22.966+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-02-01T02:05:22.978+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
